index.js

```diff
@@ -1,8 +1,5 @@
 'use strict'
 
-const PluginVirtual = require('./src/lib/plugin')
-const Token = require('./src/util/token')
-
-module.exports = PluginVirtual
-module.exports.generatePrefix = Token.prefix
-module.exports.generatePublicKey = Token.publicKey
+const Plugin = require('./src/lib/plugin')
+module.exports = Plugin.bind(null, null)
+module.exports.MakePaymentChannelPlugin = (channel) => (Plugin.bind(null, channel))
```

Comment:
that''s because b is a constructor, and get isn't a static method. If you did `new b.get()` it would work.